forloop_modules/queries/db_model_templates.py

```diff
@@ -104,12 +106,34 @@ class DeleteUidObject(BaseModel):
 
 
 class APITrigger(BaseModel):
-    trigger_name: str = ""
-    machine_uid: str = ""
-    pipeline_uid: str = ""
-    first_run_datetime: datetime.datetime
-    frequency: int = ""
-    project_uid: str = "0"
+    name: Optional[str] = None
+    first_run: datetime.datetime
+    frequency: TriggerFrequencyEnum
+    pipeline_uid: str
+    project_uid: str
+
+    @field_validator("first_run", mode="after")
+    @classmethod
+    def check_round_minutes(cls, value: datetime.datetime) -> datetime.datetime:
+        if not value.second:
+            raise ValueError("The date must have seconds set to zero.")
+        return value
+
+
+class APIDatabase2(BaseModel):
+    """
+    Cleaned up APIDatabase schema used only in ScrapingPipelineBuilders, copied to retain
+    backwards-compatibility of the old version with Desktop.
```

Comment:
Not clear why APIDatabase2 is needed from the docstring - explain better whats the difference between 1 and 2